MAINTAINERS.md

```diff
@@ -0,0 +1,9 @@
+# Maintainers
+
+Current active maintainers of this SDK:
+
+- [Grant Timmerman](https://github.com/grant)
+- [Denys Makogon](https://github.com/denismakogon)
+- [Curtis Mason](https://github.com/cumason123)
+- [Claudio Canales](https://github.com/Klaudioz)
+- [Yurii Serhiichuk](https://github.com/xSAVIKx)
```

Comment:
I'm all for adding everyone, but I guess only Claudio and I are maintaining it now

Reply:
Do you want to formally remove them as maintainers? I think a vote is needed per: https://github.com/cloudevents/spec/blob/main/docs/SDK-GOVERNANCE.md

Reply:
As I said, I don't mind people being mentioned, etc. But we can e.g. start some kind of a count-down and wait maybe half a year and see if they have any activity in the repo. What do you think?

Reply:
sounds reasonable to me

Reply:
we'll see how it goes then with this issue https://github.com/cloudevents/sdk-python/issues/222 😄